app-creator/code-editor/deserializer.js

```diff
@@ -2,32 +2,224 @@ var mapStyles = require('users/msibrahim/app-creator:map-styles');
 var sidemenu = require('users/msibrahim/app-creator:sidemenu');
 
 exports.createApp = createApp;
+exports.createResponsiveApp = createResponsiveApp;
```

Comment:
Nit: the name here is a bit confusing, I think. I'm not sure that our non-dev-focused would be able to intuit what "responsive" means. Maybe something like "createCrossPlatformApp"?

Reply:
This LGTM.